packages/application-shell/src/configure-store.ts

```diff
@@ -106,23 +109,27 @@ export const createReduxStore = (
   // additional middleware, used for testing
   additionalMiddlewares: Middleware[] = []
 ): TEnhancedStore => {
-  const store = createStore(
```

Comment:
The `createStore` method of `redux` has been deprecated and the recommended package to use is `@reduxjs/toolkit`. See https://redux.js.org/redux-toolkit/overview

The changes here are related to that.